.github/workflows/release-please.yml

```diff
@@ -49,6 +49,7 @@ jobs:
         run: echo "::set-output name=branch::${{ fromJSON(needs.release-please.outputs.pr).headBranchName }}"
       - uses: actions/checkout@v3
         with:
+          fetch-depth: 0
```

Comment:
was the `AUTHORS` build fixed then?

Reply:
Yes, I manually sorted it out last time. This will make it work consistently in CI.